### ISSUE TITLE
After updating to .net6-rc1 Blazor WebAssembly's old project is not loading and new project can't build

### ISSUE THREAD

**Issue by uzairali001** [dotnet/runtime#59156]
### Describe the bug
After updating to .net6-rc1 Blazor WebAssembly project which was working on .net6-pre7 can't be loaded in Visual studio 2022 any more and showing this error.
```
 error  : The result "" of evaluating the value "$(JsonToItemsTaskFactoryTasksAssemblyPath)" of the "AssemblyFile" attribute in element <UsingTask> is not valid.  C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.WebAssembly.Sdk\6.0.0-rc.1.21451.13\Sdk\WasmApp.Native.targets
```

Even newly created project is not building anymore
```
C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.WebAssembly.Sdk\6.0.0-rc.1.21451.13\Sdk\WasmApp.Native.targets(538,39): error MSB4022: The result "" of evaluating the value "$(JsonToItemsTaskFactoryTasksAssemblyPath)" of the "AssemblyFile" attribute in element <UsingTask> is not valid. [C:\Users\myuser\Desktop\wasm\wasm.csproj]
```

### To Reproduce

- dotnet new blazorwasm
- cd wasm
- dotnet restore
### Exceptions (if any)

```
C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.WebAssembly.Sdk\6.0.0-rc.1.21451.13\Sdk\WasmApp.Native.targets(538,39): error MSB4022: The result "" of evaluating the value "$(JsonToItemsTaskFactoryTasksAssemblyPath)" of the "AssemblyFile" attribute in element <UsingTask> is not valid. [C:\Users\myuser\Desktop\wasm\wasm.csproj]
```

### Further technical details
- ASP.NET Core version: .net6-rc1
- Include the output of `dotnet --info`
```
.NET SDK (reflecting any global.json):
 Version:   6.0.100-rc.1.21458.32
 Commit:    d7c22323c4

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22000
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.100-rc.1.21458.32\

Host (useful for support):
  Version: 6.0.0-rc.1.21451.13
  Commit:  d7619cd4b1

.NET SDKs installed:
  5.0.100 [C:\Program Files\dotnet\sdk]
  5.0.103 [C:\Program Files\dotnet\sdk]
  5.0.302 [C:\Program Files\dotnet\sdk]
  5.0.401 [C:\Program Files\dotnet\sdk]
  6.0.100-preview.4.21255.9 [C:\Program Files\dotnet\sdk]
  6.0.100-preview.6.21355.2 [C:\Program Files\dotnet\sdk]
  6.0.100-rc.1.21458.32 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.24 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.30 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.24 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.30 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.19 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.10 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.0-preview.6.21355.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.0-rc.1.21452.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.24 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.30 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.19 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0-preview.6.21352.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0-rc.1.21451.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.19 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0-preview.6.21353.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0-rc.1.21451.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version
  - Visual studio 2020 preview 4
  - VSCode 1.60.1

**Comment by radical** [comment]
@uzairali001 Is `dotnet restore` failing? Could you please run that as `dotnet restore -bl`, and share the generated `msbuild.binlog`?

**Comment by uzairali001** [comment]
`dotnet restore -bl` output:
```
C:\Program Files\dotnet\sdk\6.0.100-rc.1.21458.32\MSBuild.dll -nologo -bl -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\6.0.100-rc.1.21458.32\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Program Files\dotnet\sdk\6.0.100-rc.1.21458.32\dotnet.dll -maxcpucount -target:Restore -verbosity:m .\wasm.csproj
C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.WebAssembly.Sdk\6.0.0-rc.1.21451.13\Sdk\WasmApp.Native.targets(538,39): error MSB4022: The result "" of evaluating the value "$(JsonToItemsTaskFactoryTasksAssemblyPath)" of the "AssemblyFile" attribute in element <UsingTask> is not valid. [C:\Users\uzair\Desktop\wasm\wasm.csproj]
```
[msbuild.binlog](https://drive.google.com/file/d/1otERT4u-xZ-R4Wdn2pvFpHRwMfiF7w72/view?usp=sharing)

**Comment by radical** [comment]
Do you have `C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.MonoTargets.Sdk`? What directories does `C:\Program Files\dotnet\packs\` have?

Also, can you share your `C:\Program Files\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.json`?

And how did you install the workload?

**Comment by uzairali001** [comment]
Yes I've `C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.MonoTargets.Sdk` and it contains `6.0.0-preview.7.21377.19`
`C:\Program Files\dotnet\packs\` contains
```
Microsoft.Android.Ref
Microsoft.Android.Runtime.android-arm
Microsoft.Android.Runtime.android-arm64
Microsoft.Android.Runtime.android-x64
Microsoft.Android.Runtime.android-x86
Microsoft.Android.Sdk.BundleTool
Microsoft.Android.Sdk.win-x64
Microsoft.Android.Sdk.Windows
Microsoft.AspNetCore.App.Ref
Microsoft.iOS.Ref
Microsoft.iOS.Sdk
Microsoft.iOS.Windows.Sdk
Microsoft.MacCatalyst.Ref
Microsoft.MacCatalyst.Sdk
Microsoft.macOS.Ref
Microsoft.macOS.Runtime.osx-arm64
Microsoft.macOS.Runtime.osx-x64
Microsoft.macOS.Sdk
Microsoft.Maui.Controls.Ref.android
Microsoft.Maui.Controls.Ref.any
Microsoft.Maui.Controls.Ref.ios
Microsoft.Maui.Controls.Ref.maccatalyst
Microsoft.Maui.Controls.Ref.win
Microsoft.Maui.Controls.Runtime.android
Microsoft.Maui.Controls.Runtime.any
Microsoft.Maui.Controls.Runtime.ios
Microsoft.Maui.Controls.Runtime.maccatalyst
Microsoft.Maui.Controls.Runtime.win
Microsoft.Maui.Controls.Sdk
Microsoft.Maui.Core.Ref.android
Microsoft.Maui.Core.Ref.any
Microsoft.Maui.Core.Ref.ios
Microsoft.Maui.Core.Ref.maccatalyst
Microsoft.Maui.Core.Ref.win
Microsoft.Maui.Core.Runtime.android
Microsoft.Maui.Core.Runtime.any
Microsoft.Maui.Core.Runtime.ios
Microsoft.Maui.Core.Runtime.maccatalyst
Microsoft.Maui.Core.Runtime.win
Microsoft.Maui.Essentials.Ref.android
Microsoft.Maui.Essentials.Ref.any
Microsoft.Maui.Essentials.Ref.ios
Microsoft.Maui.Essentials.Ref.maccatalyst
Microsoft.Maui.Essentials.Ref.win
Microsoft.Maui.Essentials.Runtime.android
Microsoft.Maui.Essentials.Runtime.any
Microsoft.Maui.Essentials.Runtime.ios
Microsoft.Maui.Essentials.Runtime.maccatalyst
Microsoft.Maui.Essentials.Runtime.win
Microsoft.Maui.Extensions
Microsoft.Maui.Resizetizer.Sdk
Microsoft.NET.Runtime.Emscripten.2.0.23.Node.win-x64
Microsoft.NET.Runtime.Emscripten.2.0.23.Python.win-x64
Microsoft.NET.Runtime.Emscripten.2.0.23.Sdk.win-x64
Microsoft.NET.Runtime.MonoAOTCompiler.Task
Microsoft.NET.Runtime.MonoTargets.Sdk
Microsoft.NET.Runtime.RuntimeConfigParser.Task
Microsoft.NET.Runtime.WebAssembly.Sdk
Microsoft.NETCore.App.Host.win-arm
Microsoft.NETCore.App.Host.win-arm64
Microsoft.NETCore.App.Host.win-x64
Microsoft.NETCore.App.Host.win-x86
Microsoft.NETCore.App.Ref
Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm
Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64
Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64
Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86
Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.browser-wasm
Microsoft.NETCore.App.Runtime.Mono.android-arm
Microsoft.NETCore.App.Runtime.Mono.android-arm64
Microsoft.NETCore.App.Runtime.Mono.android-x64
Microsoft.NETCore.App.Runtime.Mono.android-x86
Microsoft.NETCore.App.Runtime.Mono.browser-wasm
Microsoft.NETCore.App.Runtime.Mono.ios-arm
Microsoft.NETCore.App.Runtime.Mono.ios-arm64
Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64
Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64
Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86
Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64
Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64
Microsoft.NETCore.App.Runtime.Mono.tvos-arm64
Microsoft.NETCore.App.Runtime.Mono.tvossimulator-arm64
Microsoft.NETCore.App.Runtime.Mono.tvossimulator-x64
Microsoft.tvOS.Ref
Microsoft.tvOS.Sdk
Microsoft.WindowsDesktop.App.Ref
NETStandard.Library.Ref
```

`C:\Program Files\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.json`
```
{
  "version": "6.0.0-rc.1.21451.13",
  "depends-on": {
    "Microsoft.NET.Workload.Emscripten": "6.0.0-rc.1.21430.3"
  },
  "workloads": {
    "wasm-tools": {
      "description": ".NET WebAssembly build tools",
      "packs": [
        "Microsoft.NET.Runtime.WebAssembly.Sdk",
        "Microsoft.NETCore.App.Runtime.Mono.browser-wasm",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.browser-wasm"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling", "microsoft-net-sdk-emscripten" ],
      "platforms": [ "win-x64", "linux-x64", "osx-x64", "osx-arm64" ]
    },
    "microsoft-net-runtime-android": {
      "abstract": true,
      "description": "Android Mono Runtime",
      "packs": [
        "Microsoft.NETCore.App.Runtime.Mono.android-arm",
        "Microsoft.NETCore.App.Runtime.Mono.android-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.android-x64",
        "Microsoft.NETCore.App.Runtime.Mono.android-x86"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling" ],
      "platforms": [ "win-x64", "linux-x64", "osx-x64", "osx-arm64" ]
    },
    "microsoft-net-runtime-android-aot": {
      "abstract": true,
      "description": "Android Mono AOT Workload",
      "packs": [
        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x86",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm64"
      ],
      "extends": [ "microsoft-net-runtime-android" ],
      "platforms": [ "win-x64", "linux-x64", "osx-x64", "osx-arm64" ]
    },
    "microsoft-net-runtime-ios": {
      "abstract": true,
      "description": "iOS Mono Runtime and AOT Workload",
      "packs": [
        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-arm64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-x64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-x86"
      ],
      "extends": [ "runtimes-ios" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "runtimes-ios": {
      "abstract": true,
      "description": "iOS Mono Runtime Packs",
      "packs": [
        "Microsoft.NETCore.App.Runtime.Mono.ios-arm",
        "Microsoft.NETCore.App.Runtime.Mono.ios-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64",
        "Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "microsoft-net-runtime-maccatalyst": {
      "abstract": true,
      "description": "MacCatalyst Mono Runtime and AOT Workload",
      "packs": [
        "Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-arm64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-x64"
      ],
      "extends": [ "runtimes-maccatalyst" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "runtimes-maccatalyst": {
      "abstract": true,
      "description": "MacCatalyst Mono Runtime Packs",
      "packs": [
        "Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "microsoft-net-runtime-macos": {
      "abstract": true,
      "description": "MacOS CoreCLR and Mono Runtime Workload",
      "packs": [
        "Microsoft.NETCore.App.Runtime.Mono.osx-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.osx-x64",
        "Microsoft.NETCore.App.Runtime.osx-arm64",
        "Microsoft.NETCore.App.Runtime.osx-x64"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling" ],
      "platforms": [ "osx-arm64", "osx-x64" ]
    },
    "microsoft-net-runtime-tvos": {
      "abstract": true,
      "description": "tvOS Mono Runtime and AOT Workload",
      "packs": [
        "Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-arm64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-arm64",
        "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-x64"
      ],
      "extends": [ "runtimes-tvos" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "runtimes-tvos": {
      "abstract": true,
      "description": "tvOS Mono Runtime Packs",
      "packs": [
        "Microsoft.NETCore.App.Runtime.Mono.tvos-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-arm64",
        "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-x64"
      ],
      "extends": [ "microsoft-net-runtime-mono-tooling" ],
      "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
    },
    "microsoft-net-runtime-mono-tooling": {
      "abstract": true,
      "description": "Shared native build tooling for Mono runtime",
      "packs": [
        "Microsoft.NET.Runtime.MonoAOTCompiler.Task",
        "Microsoft.NET.Runtime.MonoTargets.Sdk",
      ],
    }
  },
  "packs": {
    "Microsoft.NET.Runtime.MonoAOTCompiler.Task": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NET.Runtime.MonoTargets.Sdk": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NET.Runtime.WebAssembly.Sdk": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.android-arm": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.android-arm64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.android-x64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.android-x86": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x86": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86",
        "linux-x64": "Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.android-x86",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-x86",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-x86"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64",
        "linux-x64": "Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.android-x64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-x64",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-x64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm",
        "linux-x64": "Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.android-arm",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-arm",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-arm"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64",
        "linux-x64": "Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.android-arm64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-arm64",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-arm64"
      }
    },
    "Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.osx-arm64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.osx-x64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.osx-arm64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.osx-x64": {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.ios-arm" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.ios-arm64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvos-arm64",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvos-arm64",
      }
    },
    "Microsoft.NETCore.App.Runtime.Mono.tvos-arm64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
    "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-arm64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-x64" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13",
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.maccatalyst-arm64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.maccatalyst-arm64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-x64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.maccatalyst-x64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.maccatalyst-x64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvossimulator-arm64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvossimulator-arm64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-x64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvossimulator-x64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.tvossimulator-x64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.ios-arm",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.ios-arm",
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.ios-arm64",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.ios-arm64",
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-arm64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-arm64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-arm64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-x64": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-x64",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-x64"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-x86": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-x86",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.iossimulator-x86"
      }
    },
    "Microsoft.NETCore.App.Runtime.AOT.Cross.browser-wasm": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13",
      "alias-to": {
        "win-x64": "Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.browser-wasm",
        "linux-x64": "Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.browser-wasm",
        "osx-x64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.browser-wasm",
        "osx-arm64": "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.browser-wasm"
      }
    },
    "Microsoft.NETCore.App.Runtime.Mono.browser-wasm" : {
      "kind": "framework",
      "version": "6.0.0-rc.1.21451.13"
    },
  }
}
```

And I've install the workload by upgrade using visual studio installer. I've also ran `maui-check` command to update `maui` templates

**Comment by radical** [comment]
> .. `C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.MonoTargets.Sdk` and it contains `6.0.0-preview.7.21377.19`

That has preview7 for some reason. But the `WorkloadManifest.json` references:

```json
    "Microsoft.NET.Runtime.MonoTargets.Sdk": {
      "kind": "Sdk",
      "version": "6.0.0-rc.1.21451.13"
    },
```

@lewing @steveisok Could `maui-check` have mixed up the packs?

The second issue is that the import doesn't succeed but msbuild didn't complain about it. I'll file a separate bug for that.

**Comment by uzairali001** [comment]
I've fixed it by uninstalling `6.0-rc1` and deleting all `6.0.xxx` folder from `C:\Program Files\dotnet\sdk` then reinstall `6.0-rc1`

**Comment by radical** [comment]
Opened an issue for msbuild not complaining about the missing pack - https://github.com/dotnet/sdk/issues/21032 . I'm closing this since it looks like a corrupted setup. And with the sdk fix, it should correctly show that the pack isn't installed.
